server/src/schema.js

```diff
@@ -0,0 +1,68 @@
+const {gql} = require('apollo-server');
+
+const typeDefs = gql`
+    type Query {
+        events(pageSize: Int, after: String): EventConnection!
+        event(id: ID!): Event
+
+        # user information about the user currently logged in
+        me: User
+        user(id: ID!): User
+    }
+
+    type Mutation {
+        authGoogle(accessToken: AuthInput!): AuthResponse
+        authFacebook(accessToken: AuthInput!): AuthResponse
+        logout: Boolean! # success, then True.  
+    }
+
+    type AuthResponse {
+        token: String
+        name: String
+    }
+    input AuthInput {
+        accessToken: String!
+    }
+    
+    type User {
+        id: ID!
+        firstName: String!
+        lastName: String!
+        email: String!
+        age: Int
+        sex: Sex
+        city: String
+        state: String
+        phone: String
+    }
+
+    type Event {
+        id: ID!
+        host: User!
+        title: String!
+        description: String!
+        price: Float!
+        # image: Upload!  not implemented in V0
+        maxParticipants: Int!
+        tags: [Tag]!
+    }
+
+    enum Sex {
+        MAN
+        WOMAN
+        ETC # ?!?!?!
```

Comment:
Etc is not the best word for this. Please give this another thought. The above link can be helpful.

Reply:
We will think about it more. or maybe just remove gender information. Because It might be useless in our service.